secure-biometric-interface-integrator/example/react/src/App.js

```diff
@@ -0,0 +1,81 @@
+import { useEffect } from "react";
+
+import { init, propChange } from "secure-biometric-interface-integrator";
+
+const myChange = (e) => {
+  console.log("my changes");
+  console.log(e);
+};
+
+const myError = (e) => {
+  console.log("my error");
+  console.log(e);
+};
+
+const sbiEnv = {
+  env: "Staging",
+  captureTimeout: 30,
+  irisBioSubtypes: "UNKNOWN",
+  fingerBioSubtypes: "UNKNOWN",
+  faceCaptureCount: 1,
+  faceCaptureScore: 70,
+  fingerCaptureCount: 1,
+  fingerCaptureScore: 70,
+  irisCaptureCount: 1,
+  irisCaptureScore: 70,
+  portRange: "4501-4510",
+  discTimeout: 15,
+  dinfoTimeout: 30,
+};
+
+function App() {
+
+  
+  useEffect(() => {
+    init({
+      container: document.getElementById("secure-biometric-interface-integration"),
+      sbiEnv,
+      onCapture: myChange,
+      onErrored: myError,
+    });
+  }, []);
+
+  const myPropChange = () => propChange({
+    langCode: "ar",
+    disable: false,
+  })
+
+  return (
+    <div
+      className="App"
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "center",
+      }}
+    >
+      <div
+        style={{
+          margin: "4rem",
+          border: "solid red 4px",
+          padding: "20px",
+        }}
+      >
+        <div id="secure-biometric-interface-integration" style={{ width: "400px" }}></div>
```

Comment:
I see this as example but is it ok to think width as pixel and not as %?

Reply:
Yes sasi for example purpose only we have given width as "px" and not as "%".

Reply:
hi @gsasikumar, change the pixel value to %age in this [044194a](https://github.com/mosip/esignet-plugins/pull/5/commits/044194a5882341fd950e519ef9b08f59144e6caa) commit